peopleList.json

```diff
@@ -0,0 +1 @@
+[{"name":"Carlos","height":200,"hair":"black"},{"name":"almeida","height":175,"hair":"blonde"},{"name":"jose","height":175,"hair":"white"},{"name":"flavio","height":195,"hair":"black"}]
```

Comment:
This file shouldn't be commites as it's part of the runtime operation. Please add it to gitignore and delete it.

Reply:
You added to gitignore, now you need to delete it from the repo because it was pushed before you added to gitignore :)
Maybe you have to first remove from ignore, delete and then add again ;)